app/views/components/_user_intent_banner.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= tag.div class: "app-c-user_intent_banner" do %>
-  <%= tag.span title, class: "app-c-user_intent_banner__title" %>
+<%= tag.div class: "app-c-user-intent-banner" do %>
```

Comment:
Same here. This change should be fixed up to the previous commit.

Reply:
done!